Lib/test/test_dataclasses.py

```diff
@@ -1985,78 +1979,12 @@ class C:
 
         self.assertEqual(C.__doc__, "Lorem ipsum")
 
-    def test_docstring_no_fields(self):
+    def test_default_docstring(self):
         @dataclass
         class C:
             pass
 
-        self.assertDocStrEqual(C.__doc__, "C()")
```

Comment:
Why are all of these tests being deleted?

Reply:
I think we don't want to test `str(inspect.Signature)` here.